satpy/etc/readers/ici_l1b_nc.yaml

```diff
@@ -0,0 +1,650 @@
+reader:
+  name: ici_l1b_nc
+  short_name: ICI L1B RAD NetCDF4
+  long_name: EPS-SG ICI L1B Radiance (NetCDF4)
+  description: >
+    Reader for EUMETSAT EPS-SG Ice Cloud Imager Level 1B Radiance files in NetCDF4.
+  sensors: [ici]
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+
+  data_identification_keys:
+    name:
+      required: true
+    frequency_double_sideband:
+      type: !!python/name:satpy.readers.aapp_mhs_amsub_l1c.FrequencyDoubleSideBand
```

Comment:
We eventually has to adapt this once the other PR #2120 is merged. I introduced a common MW module not specific to AMSU/MHS